tests/cpp-tests/Resources/Images/nonencryptedAtlas.tps

```diff
@@ -48,7 +48,7 @@
         <key>textureSubPath</key>
         <string></string>
         <key>textureFormat</key>
-        <enum type="SettingsBase::TextureFormat">pvr2ccz</enum>
+        <enum type="SettingsBase::PixelFormat">pvr2ccz</enum>
```

Comment:
It is generated by tool, you can not modify it manually.

Reply:
*.tps is a project file generated by TexturePacker.